modules/fedimint-ln-client/src/pay.rs

```diff
@@ -169,8 +169,12 @@ impl LightningPayCreatedOutgoingLnContract {
                 }
                 Err(e) => {
                     e.report_if_important();
+
                     debug!(
-                        "Awaiting output outcome failed, retrying in {}s",
+                        error = e.to_string(),
+                        transaction_id = txid.to_string(),
+                        contract_id = contract_id.to_string(),
```

Comment:
nit: instead of `transaction_id = txid.to_string()` you can write `transaction_id = %txid`